13_time_series/13_time_series.jl

```diff
@@ -795,7 +795,14 @@ md"Well, good! We started this chapter not knowing how to tackle time series for
 
 As a final summary, when dealing with a time series it is very important to be able to define if it has any latent variables such as trend or seasonality. Once we can find that underlying information, we will be able to generate forecasts with confidence. We just need to look deeper.
 
-### Bibliography
+### Summary
+
+In this chapter we have learned the basic foundations of time series analysis. 
```

Comment:
Remember the comma after the introductory clause (I've marked this issue in several PRs):

> In this chapter, 

For more information check out https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_after_introductions.html